### DISCUSSION TITLE
clickjacking on login page

### DISCUSSION THREAD

**Discussion by hacker429** [cssbattle/forum#79] [Bugs reports]
![POC Css](https://user-images.githubusercontent.com/84594429/125191600-bf57c100-e25c-11eb-8e46-7de32410291a.PNG)
Hi,

I am security researcher and found a bug on your website,

Vulnerability: Clickjacking On Login Page

Steps To Reproduce:
1) Paste the following HTML into a text editor and save the file as .html

<html>

<head>UI REDRESSING</head>

<body>

<h1>WEBSITE IS VULNERABLE TO UI REDRESSING</h1>

<iframe width=100% height=80% src="https://cssbattle.dev/login"

 

</body>

<html>
![POC Css](https://user-images.githubusercontent.com/84594429/125191520-553f1c00-e25c-11eb-8a56-b4436b6dbd6b.PNG)

2) Open the file in a web browser.
3) Note that the iframe appears with the login page inside.

Impact:
Attacker can use this to steal victim email & password which leads to account takeover.

Screenshot is attached as a POC. If more information is needed let me know.

**Comment by chinchang** [comment]
@hacker429 Thank you for reporting! One question - but there are no passwords on this website.

**Reply by hacker429** [reply]
but is this a valid report or not?
your website is vulnerable with clickjacking.
regards
